src/util/BaseLoginController.js

```diff
@@ -100,7 +100,10 @@ define(['okta', 'q'], function (Okta, Q) {
 
     postRenderAnimation: function() {
       // Event triggered after a page is rendered along with the classname to identify the page
+      // TODO: Deprecate this event in the next major version in favor of 'render'
```

Comment:
Let's use a development mode warning to declare that this is deprecated.

Reply:
I spent a bit of time looking into doing this, but I don't think it is possible.

The Widget would need to listen to the `pageRendered` event itself, where the handler would then emit another `pageRendered` event. Of course, this introduces an endless event loop.

As long as we document the event's deprecation through our README and CHANGELOG, we should be able to encourage devs to move away from it before `3.0`.